objects/tag.c

```diff
@@ -139,7 +139,6 @@ tag_view(lua_State *L, int udx, bool view)
     {
         tag->selected = view;
         banning_need_update();
-        ewmh_update_net_current_desktop();
```

Comment:
Why we need "unfocus": Tags 1 and 2 are selected. The focused client is on tag 2, so `_NET_CURRENT_DESKTOP` has value 2. You do `client.focus = nil`. Now tag 1 is the first selected tag and `_NET_CURRENT_DESKTOP` needs to be updated.

Why we need "untagged": No client is selected. Tags 1 and 2 are selected, so the property has value 1. You deselect tag 1. Now the property should have value 2.

For accessing `tag_class`: Perhaps it is easier to move it into `objects/tag.h`, just like `objects/client.h` contains `client_class`.